rasa/core/policies/rule_policy.py

```diff
@@ -202,12 +190,35 @@ def predict_action_probabilities(
             )
 
         if possible_keys:
+            # TODO rethink that
             key = max(possible_keys, key=len)
 
             recalled = self.lookup.get(key)
+
+            if active_form_name:
+                predicted_listen_from_general_rule = recalled is None or (
```

Comment:
```suggestion
                # Check if a FAQ rule was applied after a rejection. 
                # FAQ rules might not explicitly switch back to the `Form`.
                # Hence, we have to take care of that.
                predicted_listen_from_general_rule = recalled is None or (
```

Reply:
I changed your comment to be any rule, because using FAQ here sounds too specific